src/store/transactions/transfer/transfer.thunks.ts

```diff
@@ -133,11 +138,82 @@ function fetchFees(): AppThunk {
   };
 }
 
+function checkTxData(txData: TxData) {
+  return (dispatch: AppDispatch): void => {
+    const { amount, from, to, feesTask } = txData;
+
+    if (isHermezBjjAddress(txData.to)) {
+      void CoordinatorAPI.getAccounts(to, [from.token.id]).then(
```

Comment:
This is an [ESLint rule](https://github.com/typescript-eslint/typescript-eslint/blob/main/packages/eslint-plugin/docs/rules/no-floating-promises.md) which I think is very useful, because with the `void` operator you easily know that you are not doing anything with the return value of the `Promise`.